src/lib/config/config.ts

```diff
@@ -6,7 +6,8 @@ export const kindeConfiguration = {
 	authDomain: env.KINDE_ISSUER_URL,
 	clientId: env.KINDE_CLIENT_ID,
 	logoutRedirectURL: env.KINDE_POST_LOGOUT_REDIRECT_URL,
-	redirectURL: env.KINDE_REDIRECT_URL,
+	redirectURL: env.KINDE_SITE_URL + '/api/auth/kinde_callback',
```

Comment:
This is a breaking change for your customers.

1️⃣ It requires me to update my production configuration with extra care. I also have a custom Vite plugin so I can sign in from Netlify deploy preview URLs that would need to change. Not preferred :)

2️⃣ This introduces a new hard requirement for the Kinde callback to be hosted under the route `/api/auth/kinde_callback`. Previously it was possible to host (intended or not) under eg. `/auth/kinde_callback`. I moved my routing, but you could consider it a breaking change that needs to be documented.

You could keep the redirect URL variable, and extract the `appBase` base URL using `new URL(redirectURL).origin`. That way I keep my routing, it's a smooth upgrade for your customers and no potential for breakage :)

I prefer not having `/api` in my front-end route, but it's not a dealbreaker for me.

Reply:
Hi @rjkip 
Thanks for your review, I think so, that is an efficient way to do it.

Reply:
Cheers, that looks good. I'll try and test tomorrow or Monday by the latest. Having this work for registration too would be great 🙏